tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -355,6 +355,7 @@
         - configure-os
         - run-os
         - install-certs
+        - ceph-hci-pre
```

Comment:
+1 from me, if Ceph is part of the deployment we need the firewall in place. However I think this might apply to standalone only, because in multinode we're not going to enroll the Controllers as edpm nodes, right? In any case it doesn't hurt, I'm just wondering if we want to put a comment to explain why we need that and which is the use case / scenario we're covering.